app/assets/stylesheets/components/_intervention.scss

```diff
@@ -0,0 +1,3 @@
+.gem-c-intervention {
```

Comment:
We can remove this file if the approach mentioned in `application.html.erb` is used